src/EditorFeatures/Core/Preview/AbstractPreviewFactoryService.cs

```diff
@@ -69,17 +67,17 @@ internal abstract class AbstractPreviewFactoryService<TDifferenceViewer> : Foreg
             _previewRoleSet = previewRoleSet;
         }
 
-        public SolutionPreviewResult GetSolutionPreviews(Solution oldSolution, Solution newSolution, CancellationToken cancellationToken)
+        public SolutionPreviewResult GetSolutionPreviews(Solution oldSolution, Solution? newSolution, CancellationToken cancellationToken)
```

Comment:
what does it mean to have a solution diff preview when the new solution is null?

Reply:
I have no idea either. It was null checked, so clearly it's allowed.

Reply:
(this was me very much annotating the implied state of the code, but not asking why)